### PR TITLE
modify grafana chart as prometheus like format

### PR DESCRIPTION
**Pull request by zanhsieh**
1. Prometheus-like chart format
2. Dashboard example
3. Initial add datasource example

### CODE REVIEW

File: incubator/grafana/Chart.yaml [hunk 1]
```diff
@@ -1,3 +1,10 @@
-description: A Helm chart for Kubernetes
 name: grafana
-version: 0.1.3
+version: 0.1.2
```
Comment (viglesiasce) [inline incubator/grafana/Chart.yaml]: Please bump the version to 0.2.0

File: incubator/grafana/values.yaml [hunk 2]
```diff
@@ -1,11 +1,1039 @@
...
+  ##
+  resources:
+    # limits:
+    #   cpu: 500m
+    #   memory: 512Mi
+    requests:
+      cpu: 100m
+      memory: 100Mi
+
+  serviceType: NodePort
```
Comment (viglesiasce) [inline incubator/grafana/values.yaml]: Make this a  ClusterIP by default

File: incubator/grafana/values.yaml [hunk 3]
```diff
@@ -1,11 +1,1039 @@
...
+
+    [grafana_net]
+    url = https://grafana.net
+
+
+## Grafana dashboard files ConfigMap entries
+## Use prometheus and install on the same namespace as example.
+##
+serverDashboardFiles:
+  prometheus.json: |
```
Comment (viglesiasce) [inline incubator/grafana/values.yaml]: Please remove the default dashboard as they will not work out of the box.

**Review by viglesiasce — CHANGES_REQUESTED** [review]
Thanks for this @zanhsieh!

Can you also add a NOTES.txt that gives users an idea of how to connect to grafana once its installed. Checkout out the stable/jenkins/templates folder for an example. 

Once the NOTES.txt is in there, this chart is viable for the stable repo. Can you move it over to that folder?

A few other things to change here as well

**Comment by zanhsieh** [comment]
@viglesiasce Modified. Please review. Thanks.

File: stable/grafana/templates/NOTES.txt [hunk 4]
```diff
@@ -0,0 +1,39 @@
+1. Get your '{{ .Values.server.adminUser }}' user password by running:
+
+   printf $(printf '\%o' `kubectl get secret --namespace {{ .Release.Namespace }} {{ template "server.fullname" . }} -o jsonpath="{.data.grafana-admin-password[*]}"`);echo
```
Comment (viglesiasce) [inline stable/grafana/templates/NOTES.txt:3]: Just a note that this doesnt appear to work with kubectl 1.5. Lets not change anything quite yet until we figure out a good solution that we can apply to the other charts.

**Review by viglesiasce — APPROVED** [review]

**Comment by viglesiasce** [comment]
Thanks @zanhsieh!!

**Comment by viglesiasce** [comment]
@k8s-bot ok to test

**Comment by viglesiasce** [comment]
Test failure is due to the fact that files were removed then readded rather than placed with a `git mv`.

**Pull request merged**